packages/performance/src/performance.ts

```diff
@@ -6,7 +6,7 @@ import {
   withNavigation,
   withTiming,
   supportsPerformanceObserver,
-  referenceTime,
+  timeOrigin as referenceTime,
```

Comment:
Because I don't think we ever exported `referenceTime` from `@shopify/performance`, you can probably just export `timeOrigin` here.

Reply:
This is an import, not the export of the util. The `timeOrigin` util was previously named `referenceTime` but we're renaming it in this PR. The variable in this file that uses `referenceTime` is `const timeOrigin = referenceTime()`. 

So I used the `as referenceTime` renaming to preserve the other naming in this file, but maybe we could improve it? Would something like `crossBrowserTimeOrigin` or `browserSafeTimeOrigin` be better a better name alias? Or maybe we'd prefer to rename the `timeOrigin` variable?